### PR TITLE
Added Jon Todd as an author

### PR DESCRIPTION
**Pull request by jtodd-okta**
@joelfranusic-okta Can you take a look?

### CODE REVIEW

**Comment by joelfranusic-okta** [comment]
Looks good to me!

File: README.md [hunk 1]
```diff
@@ -85,5 +85,5 @@ title: TITLE_CASE_NAME_OF_SECTION
 
 ### Authors
 
-1.  Create an entry in _data/authors.yml
+1.  Create an entry in _sources/_data/authors.yml
```
Comment (joelfranusic-okta) [inline README.md]: Nice, thanks!

**Comment by zdevil** [comment]
Looks good but the image should also be added under _source.

**Comment by jtodd-okta** [comment]
Thanks Hamza, I realized that after pushing. The next commit will move this to the correct spot.


~ Jon

________________________________

**Pull request merged**